cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -256,8 +257,9 @@ public boolean installAsDependency(VersionRange version, ProcessContext processC
                 + " and this tool does not support the software repository.");
       }
       this.context.info(
-          "Configured version of tool {} is {} but does not match version to install {} - need to use different version from software repository.",
-          this.tool, configuredVersion, version);
+          "The tool {} requires {} in the version range {}, but your project uses version {}, which does not match."
+              + " Therefore, we install a compatible version in that range.",
+          toolParent, this.tool, version, configuredVersion);
```

Comment:
In my review summary I wrote:
> Could you please add the story https://github.com/devonfw/IDEasy/issues/1213 to our CHANGELOG?

See [DoD](https://github.com/devonfw/IDEasy/blob/main/documentation/DoD.adoc)

In order that it does not get forgotten, I add it as a review comment to an arbitrary line of code.